File: core/src/com/symbol/ecs/component/JumpComponent.java
package com.symbol.ecs.component;

import com.badlogic.ashley.core.Component;
import com.badlogic.gdx.utils.Pool;

public class JumpComponent implements Component, Pool.Poolable {

    public float impulse = 0f;

    @Override
    public void reset() {
        impulse = 0f;
    }

}

File: core/src/com/symbol/ecs/component/JumpComponent.kt
package com.symbol.ecs.component

import com.badlogic.ashley.core.Component
import com.badlogic.gdx.utils.Pool

class JumpComponent : Component, Pool.Poolable {

    var impulse = 0f

    override fun reset() {
        impulse = 0f
    }

}